source/by-role/system-admin/upgrade/upgrading-search.rst

```diff
@@ -54,35 +54,103 @@ Here is the ``upgrade-search`` script params:
 Here are the steps to  upgrade your CrafterCMS  3.1.x install:
 
 #. Download the CrafterCMS version you'd like to upgrade to, and extract the files.
+#. Configure the root directory of the new bundle to use Java version 17 (This is required to run the ``upgrade-search.sh`` 
+   script since ES 7 won't run with Java 21)
 #. Run the ``upgrade-search.sh`` script from your newly extracted files.
 
    .. code-block:: bash
        :caption: *Run the upgrade-search script*
-
-       ./upgrade-search.sh /path/of/install/to/be/upgraded --stay-alive
-
-#. Upgrade using the ``upgrade-target.sh`` script
+       :emphasize-lines: 1
+
+       ./bin/upgrade/upgrade-search.sh /path/of/install/to/be/upgraded --stay-alive
+       ========================================================================
+       Search upgrade started
+       ========================================================================
+       ...
+       End process. Stop Elasticsearch
+       Move indexes from 'data/indexes-es' to 'indexes'
+       ========================================================================
+       Search upgrade completed
+       ========================================================================
+
+#. Configure the root directory back to Java version 21
+#. Upgrade using the ``upgrade-target.sh`` script. We'll need to upgrade the target without backing up the bin folder.
+   This is because the MariaDB version is now 11 for 4.2.0, and we cannot start the old MariaDB from the new bundle.
+   Please manually back it up before running the ``upgrade-target.sh`` script. Remember to enter ``No`` when prompted
+   by the script ``Backup the bin folder before upgrade? [(Y)es/(N)o]:``
 
    .. code-block:: bash
        :caption: *Run the upgrade-target script*
+       :emphasize-lines: 1,3
 
-       ./upgrade-target.sh /path/of/install/to/be/upgraded
+       ./bin/upgrade/upgrade-target.sh /path/of/install/to/be/upgraded
+       ...
+       > Backup the bin folder before upgrade? [(Y)es/(N)o]:n
+       ...
+       ========================================================================
+       Upgrade completed
+       ========================================================================
+       !!! Please read the release notes and make any necessary manual changes, then run the post upgrade script: /path/of/install/to/be/upgraded/bin/upgrade/post-upgrade.sh !!!
 
-#. Run the ``post-upgrade.sh`` script from the install that's being upgraded.  Before starting CrafterCMS, you'll need to configure the installation root directory to use Java version 17.  Remember to read the release notes or any relevant upgrade articles and make any necessary manual changes before running the `post-upgrade.sh`` script
+#. Before performing the post-upgrade, please upgrade the DB in the install that's being upgraded.
```

Comment:
@jmendeza isn't the `upgradedb` now removed from the upgrade scripts and done automatically? related to https://github.com/craftersoftware/craftercms/issues/1010

Reply:
Yes @yacdaniel, DB upgrade would happen on startup.